Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/formatter/FormatterHelper.kt

```diff
@@ -37,12 +17,11 @@ fun formatDrawable(value: Boolean, drawableTrue: Int, drawableFalse: Int): Drawa
  * @param colorFalse
  * @return
  */
-fun formatColor(value: Boolean, colorTrue: Int, colorFalse: Int): Int {
-    val appContext = CoronaWarnApplication.getAppContext()
+fun formatColor(context: Context, value: Boolean, colorTrue: Int, colorFalse: Int): Int {
```

Comment:
Could make it an extention function, e.g. Context.formatColor

Reply:
Not sure about this..
Methods of Formatter are meant to be used in layouts and extension funs can't be used there